### DISCUSSION TITLE
Unable to find rke2-airgap-images tarball artifact

### DISCUSSION THREAD

**Discussion by miket1** [rancher/rke2#1843] [Q&A]
The rke2 installation guide says:
RKE2 can be installed in an air-gapped environment with two different methods. You can either deploy via the rke2-airgap-images tarball release artifact, or by using a private registry.

However, when I go to the Assests for the latest release there is no tarball listed for rke2-airgap-images. Could you please help point me in the right direction? Thank you!

**Comment by brandond** [comment]
Did you scroll further down that page to see: https://docs.rke2.io/install/airgap/#tarball-method ? This tells you which specific files to download.

**Comment by miket1** [comment]
Not quite. it talks about downloading rke2-images.linux-amd64.tar.zst, or rke2-images.linux-amd64.tar.gz for releases prior to v1.20. Since we are on releases greater than 1.20, the assumption i made was to download the airgap tarball (see 1st paragraph) of the airgap install section (https://docs.rke2.io/install/airgap/), but this rke2-airgap-image tarball is not found under assets and thus there's confusion. The 1st paragraph makes it seem like there's a special tarball available (rke2-airgap-image) for airgap installation. Is that not the case? Thanks

**Reply by brandond** [reply]
I see how that wording could be a bit ambiguous. The "rke2-airgap-images tarball release artifact" referenced in that paragraph is one or more of the compressed tarballs listed in the detailed instructions of the "Tarball Method" section. Only the first item in that detailed list includes the full extension; the extension is omitted for the sake of brevity on subsequent list items.
